### PR TITLE
converted notice to event

### LINKED ISSUE #93: convert notice to event
### Description
In the server rename the notices routes and schemas to events and delete the existing events routes and schemas.

Remember to checkout backend and make PR to the backend branch!

### Screenshots/testimonials
Notice routes: https://github.com/houseofgeeks/hg/blob/back-end/backend/routes/notice.js

### PR DESCRIPTION
**Pull request by ankiiitraj**
fixes #93 
Converted Notice to events!
#### Checklist

- [x] I've followed the Contributing guidelines provided in the repository.
- [x] I've made the changes that were demanded in the linked issue.
- [x] I've tested my code on a Chromium-based browser.
- [x] I've tested my code on Mozilla Firefox.
- [x] My code gave a clean console on debugging. (no warnings/errors)

#### Testimonial
<provide a relevant screenshot or any testimonial for the solved issue if applicable>
N/A

### CODE REVIEW

**Comment by ankiiitraj** [comment]
Thoroughly review this PR, might contain some mistakes due to the renaming of files.

File: client/src/components/Home/AboutUs/Event/Event.js [hunk 1]
```diff
@@ -3,56 +3,52 @@ import axios from "axios";
...
 import mobilePlanet from "./mobile-planet.svg";
-import "./Notice.css";
+import "./Event.css";
 
-function Notice() {
-  const [notices, setNotice] = useState([
-    { title: "fetching notices", description: " . . ." },
+function Event() {
+  const [events, setEvent] = useState([
+    { title: "Recieving signals...", description: "Decrypting... Latest News..." },
```
Comment (EmperorYP7) [inline client/src/components/Home/AboutUs/Event/Event.js]: We can leave it the same. `{ title: "fetching events", description: " . . ." }`

File: client/src/components/Home/AboutUs/Event/Event.js [hunk 2]
```diff
@@ -3,56 +3,52 @@ import axios from "axios";
...
   }, []);
 
   return (
-    <div className="notice-section">
-      <div className="notice-title">From the Geeks' Satellite..</div>
-      <div className="notice-content">
-        <div className="title">
-          Recieving signals...{" "}
-          <span className="decrypting"> Decrypting...</span>
-          <span className="latest-news"> Latest News...</span>
```
Comment (EmperorYP7) [inline client/src/components/Home/AboutUs/Event/Event.js:36]: I think we should maintain this structure.. and replace "Latest News..." with "Latest Events.."

**Review by EmperorYP7 — CHANGES_REQUESTED** [review]

File: client/src/components/Home/AboutUs/Event/Event.js (hunk 1 above)
Comment (anktrj) [reply client/src/components/Home/AboutUs/Event/Event.js]: done!

**Review by tusharjain0022 — APPROVED** [review]
LGTM! 💯

**Review by hariom1625 — APPROVED** [review]
LGTM 🙌

**Review by EmperorYP7 — APPROVED** [review]
LGTM!

**Pull request merged**